modules/navier_stokes/src/kernels/INSADMomentumCoupledForce.C

```diff
@@ -49,19 +49,10 @@ INSADMomentumCoupledForce::INSADMomentumCoupledForce(const InputParameters & par
       _fe_problem.getUserObject<INSADObjectTracker>("ins_ad_object_tracker"));
   obj_tracker.set("has_coupled_force", true);
   if (has_coupled)
-  {
-    std::vector<const ADVectorVariableValue *> coupled_vars;
-    for (const auto i : make_range(coupledComponents("coupled_vector_var")))
-      coupled_vars.push_back(&adCoupledVectorValue("coupled_vector_var", i));
-    obj_tracker.set("coupled_force_var", coupled_vars);
-  }
+    obj_tracker.set("coupled_force_var", getParam<std::vector<VariableName>>("coupled_vector_var"));
```

Comment:
Not going to list all.

Reply:
I feel like this one is fine right? I don't see any `Coupleable` APIs that return the whole vector of coupled variables given a name